packages/components/psammead-image/src/index.jsx

```diff
@@ -21,13 +37,15 @@ export const Img = ({ alt, src, srcset, height, width }) => {
 
 Img.propTypes = {
   alt: string.isRequired,
+  fade: bool,
   src: string.isRequired,
   srcset: string,
   height: oneOfType([string, number]),
   width: oneOfType([string, number]),
 };
 
 Img.defaultProps = {
+  fade: true,
```

Comment:
Do we want fade enabled by default? This is most likely a question for UX // @nataliesmart